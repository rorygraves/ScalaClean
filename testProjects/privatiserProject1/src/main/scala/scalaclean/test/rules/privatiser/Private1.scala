package scalaclean.test.rules.privatiser.p1

object Private1 extends App {
  // some private[this] elements
  // can't detect [this] though so should be private
  // where it isnt [this] already

  val a = 1
  def b = 2
  var c = 3
  a + b + c

  private[Private1] val d = 1
  private[test] def e     = 2
  private var f           = 3
  private[this] def h     = 3
  d + e + f + h

  protected[Private1] val l = 1
  protected[test] def m     = 2
  protected var n           = 3
  protected[this] def o     = 3
  l + m + n + o
}
