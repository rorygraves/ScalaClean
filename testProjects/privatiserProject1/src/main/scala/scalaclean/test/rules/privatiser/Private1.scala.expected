package scalaclean.test.rules.privatiser.p1

object Private1 extends App {
  // some private[this] elements
  // can't detect [this] though so should be private
  // where it isnt [this] already

  private val a = 1
  private def b = 2
  private var c = 3
  a + b + c

  private val d = 1
  private def e     = 2
  private var f           = 3
  private[this] def h     = 3
  d + e + f + h

  private val l = 1
  private def m     = 2
  private var n           = 3
  protected[this] def o     = 3
  l + m + n + o
}
